luci/luci-app-falter-owm/luasrc/controller/antennas.lua

```diff
@@ -16,8 +16,8 @@ $Id$
 module("luci.controller.antennas", package.seeall)
 
 function index()
-	local page = entry({"admin", "system", "antennas"}, cbi("antennas"), "Antennas settings", 10)
-	page.dependent = true	
-	assign({"mini", "system", "antennas"}, {"admin", "system", "antennas"}, "Antennnas settings", 1)
+	local page = entry({"admin", "system", "antennas"}, cbi("antennas"), _("Antennas settings"), 10)
+	page.dependent = true
+	assign({"mini", "system", "antennas"}, {"admin", "system", "antennas"}, _("Antennnas settings"), 1)
```

Comment:
According to the [luci-wiki](https://github.com/openwrt/luci/wiki/i18n) the underscore-function works in new-style-js app only. For lua we should keep the `translate()` function.

Reply:
Using _("text") is how we have it in https://github.com/Freifunk-Spalter/packages/blob/master/luci/luci-mod-falter/luasrc/controller/freifunk/freifunk.lua so it should work fine.

Reply:
Here is an example from the luci repo (from openwrt-19.07 branch) which also uses this style...

https://github.com/openwrt/luci/blob/openwrt-19.07/modules/luci-mod-network/luasrc/controller/admin/network.lua